agr_java_core/src/main/java/org/alliancegenome/cache/repository/helper/AlleleFiltering.java

```diff
@@ -158,7 +158,7 @@ public AlleleFiltering() {
         filterFieldMap.put(FieldFilter.PHENOTYPE, phenotypeFilter);
         filterFieldMap.put(FieldFilter.DISEASE, diseaseFilter);
         filterFieldMap.put(FieldFilter.VARIANT_TYPE, variantTypeFilter);
-        filterFieldMap.put(FieldFilter.VARIANT_CONSEQUENCE, variantConsequenceFilter);
+        filterFieldMap.put(FieldFilter.MOLECULAR_CONSEQUENCE, variantConsequenceFilter);
```

Comment:
Sorry stickler for naming, `variantConsequenceFilter` -> `molecularConsequenceFilter`?

And there are a bunch of others.